Artifacts.toml

```diff
@@ -1,16 +1,19 @@
+[ameriflux_data_US-MOz]
```

Comment:
not sure this file should change

Reply:
uh I am not sure what happened - I'll remove this